src/jackdaw/serdes/avro.clj

```diff
@@ -525,8 +525,14 @@
                                 (catch clojure.lang.ExceptionInfo e
                                   (let [data (-> e
                                                  ex-data
-                                                 (assoc :topic topic :clj-data data))]
-                                    (throw (ex-info (.getMessage e) data))))))}
+                                                 (assoc :topic topic :clj-data data))
```

Comment:
Appreciate it was already in the previous version of the code, but can we rename either the left or right hand-side `data` to remove the variable shadowing?

Reply:
👍